apps/ideaspace/src/components/modules/IdeaSubmissionTC/IdeaSubmissionTC.js

```diff
@@ -0,0 +1,186 @@
+import React from 'react';
+import axios from "axios";
+import { useUserDataContext } from '@devlaunchers/components/context/UserDataContext';
+import { atoms } from '@devlaunchers/components/src/components';
+import ReactMarkdown from 'react-markdown';
+import remarkGfm from 'remark-gfm';
+import { markdownSource, quickLinkContentLeft, quickLinkContentRight } from './MarkdownIdeaspaceTC';
+import BackButton from '../../common/BackButton/BackButton';
+
+import {
+  QuickLinkLeft,
+  Content,
+  QuickLinkTop,
+  MiddleSizeQL,
+  SmallSizeQL,
+  BackButtonArea,
+  Dropdown,
+  HeadWapper,
+  Headline,
+  StyledRanbow,
+  QuickLinkMiddle,
+  MarkdownWrapper,
+} from './StyledIdeaSubmissionTC';
+
+let scrollElementOffset = 0;
```

Comment:
Does this need to be global? Can we make it a stateful variable within our component?